README.md

```diff
@@ -4,7 +4,9 @@
 
 This Project consists of making a calculator with the goal of solving transcendental functions without using existing math libraries.
 
-Information on team organization, tech stack, and the user guide can be found on [our wiki](https://github.com/neoJINXD/Eternity/wiki).
+Information on team organization and tech stack can be found on [our wiki](https://github.com/neoJINXD/Eternity/wiki).
+
+The user guide can be found under [User_Guide.pdf](https://github.com/neoJINXD/Eternity/User_Guide.pdf).

```

Comment:
Assuming this url will work after we push

Reply:
Yup it will work once we push to master, it's on my list to test once we do that